unit_tests/src/test_procApiRequest_raw.cpp

```diff
@@ -137,26 +147,20 @@ TEST_CASE("#8653: procApiRequest_raw allows overriding packing instructions")
                                            pi_name_out,
                                            static_cast<void**>(static_cast<void*>(&output)),
                                            nullptr);
-        REQUIRE(ec == COLL_OBJ_T);
-
-        // We expect to get the size and type back. For a collection, the size is always
-        // zero. One of the primary pieces of info we're interested in is the type. That
-        // should be extracted and stored in the output data structure.
-        CHECK(0 == output->objSize);
-        CHECK(COLL_OBJ_T == output->objType);
-
-        // The remaining member variables likely contain garbage because the data was
-        // never extracted from the server's response. The reason for this is because the
-        // packing instruction does not mention those members. This behavior cannot be
-        // asserted because the output pointer will point to a heap-allocated object. We
-        // cannot know what the initial state of the heap-allocated is.
-        //
-        // With that said, the behavior has been observed to be true through manual testing.
+        REQUIRE(ec == SYS_PACK_INSTRUCT_FORMAT_ERR);
     }
 
-    SECTION("bad input packing instruction")
+    // The following code is disabled because the test for truncated input packing
+    // instructions can fail unexpectedly, causing testing to take more time than needed.
+    // This behavior has not been observed for the truncated output packing instruction
+    // test.
+    //
+    // The other reason for disabling these tests is due to there being no known use-case
+    // for truncated packing instructions.
```

Comment:
Should we document the behavior of truncated input packing instructions as undefined if it is known to cause issues? I see the following in the docstring for `procApiRequest_raw`:
> ```
> /// \warning This function is provided for handling changes in packing instructions for an API
> /// only. All other properties of the original API must be honored. The behavior of this function
> /// is undefined if preconditions are violated.
> ```

Is "preconditions" referring to the truncated input packing instructions, or something else?

Reply:
Okay. So the idea is that the packing instructions should always include all of the members required by the API, which is kind of a given?

Reply:
That's correct. The only focus of the new function is using custom packing instructions. Everything else remains the same.